pkg/cephfs/util.go

```diff
@@ -49,38 +49,39 @@ func closePipeOnError(pipe io.Closer, err error) {
 
 func execCommand(program string, args ...string) (stdout, stderr []byte, err error) {
 	cmd := exec.Command(program, args...) // nolint: gosec
-	klog.V(4).Infof("cephfs: EXEC %s %s", program, util.StripSecretInArgs(args))
+	stripArgs := util.StripSecretInArgs(args)
```

Comment:
eventhough its looks ok, I dont find some value for extra variable here with `stripArgs`. We could directly use util function in the return call. @gman0 thoughts ?

Reply:
@humblec main idea to add a variable to avoid computing time, I didn't want to call `util.StripSecretInArgs()` multiple times inside same  function.

Reply:
@Madhu-1 not sure I get it, can you explain how it helps in avoiding computing time ?. 
> I didn't want to call `util.StripSecretInArgs()` multiple times inside same function.
>
see, either we keep writing `util.StripSecretInAgrs()` multiple time or we write `stripArgs` multiple time. I dont see much value here. But I am open for other thoughts.